spec/support/shared/examples/metasploit/cache/module/descendant.rb

```diff
@@ -0,0 +1,17 @@
+shared_examples_for 'Metasploit::Cache::Module::Descendant' do |ancestor:, factory:|
+  context 'associations' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.